### DISCUSSION TITLE
Luo Meizheng quest impossible with No Fungal Monsters?

### DISCUSSION THREAD

**Discussion by MilosRasic** [CleverRaven/Cataclysm-DDA#46921] [General]
So, she wants a tainted veggie. Is there an alternate source of these or would it make sense to consider an alternate quest if No Fungal Monsters is enabled?

**Comment by actual-nh** [comment]
Looks like you _might_ find some in an ant colony, judging by `ant_food` in data/json/itemgroups/Monsters_Animals_Lairs/monster_drops_lairs.json. (Or in a mutagen lab.) Neither is by any means certain, of course!
